include/ensmallen_bits/aug_lagrangian/aug_lagrangian.hpp

```diff
@@ -49,13 +48,24 @@ class AugLagrangian
    *     class.
    * @param function The function to optimize.
    * @param coordinates Output matrix to store the optimized coordinates in.
+   * @param penaltyThresholdFactor When the penalty threshold is updated set
+   *    the penalty threshold to the penalty multplied by this factor. The
+   *    default value of 0.25 is is taken from Burer and Monteiro (2002).
+   * @param sigmaUpdateFactor When sigma is updated  multiply sigma by this
+   *    value. The default value of 10 is taken from Burer and Monteiro (2002).
```

Comment:
It's really nice to provide customizable values for these.  My experiences working with the AugLagrangian optimizer when solving LRSDPs indicated that it can be really sensitive to these parameters, but for whatever reason I did not make them customizable when I wrote it.  So this is a nice improvement. :+1: